lmdeploy/pytorch/config.py

```diff
@@ -75,7 +89,7 @@ class ModelConfig:
     num_attention_heads: int
     num_key_value_heads: int
     bos_token_id: int
-    eos_token_id: int
+    eos_token_id: List[int]
```

Comment:
AFAK, there is only one eos_token_id for each model.

Reply:
There are often multiple EOS tokens, and it has tripped up HF transformers as well.  E.g. llama-3 has eot and eos and both need to be stopped on.  Others are same.

Reply:
Would it be one eos_token_id and multiple stop words?

Reply:
It's identified as 2 eos_token_ids.  HF added proper support for this after llama-3 and other models required it.

https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct/blob/main/generation_config.json#L3